tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -54,12 +54,13 @@ internal TracerManager CreateTracerManager(ImmutableTracerSettings settings, Tra
                 sampler: null,
                 scopeManager: previous?.ScopeManager, // no configuration, so can always use the same one
                 statsd: null,
-                runtimeMetrics: null,
+                runtimeMetrics: previous?.RuntimeMetrics,
```

Comment:
I don't think we can do this: `RuntimeMetricsWriter` depends on statsd, and statsd may need to change here (changing exporter settings in code). Why did you need to change this for your PR?